src/styles/blocks/card.scss

```diff
@@ -0,0 +1,124 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 944px;
+  margin: 0 auto;
+  padding: 50px 0;
+
+  &__list {
+    margin-bottom: 46px;
+
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    align-content: center;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__item {
+    max-width: 200px;
+    display: flex;
+    flex-direction: column;
+    border-radius: 5px;
+    background-color: #fff;
+    border: 1px solid #f3f3f3;
+    font-size: 10px;
+    line-height: 14px;
+    padding: 32px 16px 16px;
+    color: #060b35;
+    margin-bottom: 46px;
+  }
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__image-img {
+    height: 134px;
+    width: 160px;
+  }
+
+  &__product-name {
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    letter-spacing: 0;
+    margin-bottom: 4px;
+  }
+
+  &__product-id {
+    font-family: "Roboto", sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: 0;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-txt {
+    font-family: "Roboto", sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: 0;
+    align-self: flex-end;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-amount {
+    font-family: "Roboto", sans-serif;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    letter-spacing: 0;
+  }
+
+  &__button-link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: rgb(0, 172, 220);
```

Comment:
Instead of using a hardcoded color code, consider using a variable for the background-color.